portalnet/src/overlay/protocol.rs

```diff
@@ -750,7 +750,7 @@ where
                     successfully_bonded_bootnode = true;
                 }
                 Err(err) => {
-                    error!(
+                    warn!(
```

Comment:
```suggestion
                    debug!(
```

I think we should only print a error if we fail connecting to all bootnodes,

A warn would still show this message every time on boot, which 1 bootnode failing isn't that note-able

^ this view is a discussion

Reply:
Yeah... my only question is that I still do find it pretty useful to know which bootnodes are failing... for example, right now trin's are failing. Which, otherwise I wouldn't know if we're still able to connect to a fluffy node. So I think this case is still worthy of a `warn!` level...

Reply:
In that case `warn!()` is fine for now and we can think about moving it to debug!() in the future when things are more stable

Reply:
> Maybe we could just compile a list of all "failed" bootnodes... something like... `warn!("Failed to connect to bootnodes: trin-ams3-1, fluffy1")` so that it's just a single warn log rather than many

I really like this idea

Reply:
> Maybe we could just compile a list of all "failed" bootnodes... something like... `warn!("Failed to connect to bootnodes: trin-ams3-1, fluffy1")` so that it's just a single warn log rather than many

Maybe report the number of successful bootnodes connected to as well then, to make it not appear like things are only failing